docs/openapi/api-internal-b2b-pa-v1.yaml

```diff
@@ -0,0 +1,575 @@
+          description: >-
+            Identificativo univoco all'interno della request HTTP, serve per correlare la risposta. 
+          type: string
+        contentType:
+          title: MIME type del file che verrà caricato
+          description: >-
+            Il MIME type dell'allegato che dovrà essere caricato. Attualmente è supportato solo
+              - application/pdf
+          type: string
+          
+    PreLoadResponse:
+      title: Informazioni utili al caricamento file
+      description: >-
+        Per ogni richiesta che è stata fatta viene fornito un presigned URL e le 
+        informazioni per usarlo.
+      type: object
+      properties:
+        preloadIdx:
+          description: per correlazione con la richiesta
+          type: string
+        secret:
+          description: >-
+            Token aggiuntivo per far si che sia necessario intercettare anche gli 
+            header e non solo l'URL di upload del contenuto del documento.
+          example: AZ23RF12
+          type: string
+        httpMethod:
+          description: >-
+            Indica se per l'upload del contenuto file bisogna utilizzare il metodo PUT o POST
+          type: string
+          enum:
+            - POST
+            - PUT
+        url:
+          description: >-
+            URL a cui effettuare l'upload del contenuto del documento.
+          example: 'https://preloadpn.aws.amazon.......'
+          type: string
+        key:
+          description: >-
+            la chiave restituita sarà globalmente unica e andrà utilizzata nella richiesta 
+            di notifica.
+          example: '8F7E/9A3B/1234/AB87'
+          type: string
+    
+
+    ###########################################################################################
+    ###                              DTO RICHIESTE DI NOTIFICA                              ###
+    ###########################################################################################
+
+    NewNotificationResponse:
+      title: Identificativi della richiesta di notifica
+      description: >-
+        Contiene le informazioni utili per identificare una richiesta di invio notifica
+        che non è ancora stata accettata da Piattaforma notifiche.
+      type: object
+      required:
+        - notificationRequestId
+        - paProtocolNumber
+      properties:
+        notificationRequestId:
+          type: string
+          description: >-
+            identificativo univoco di una richiesta di invio notifica, non è lo IUN
+        paProtocolNumber:
+          type: string
+          description: Identificativo inviato dalla pubblica amministrazione
+        idempotenceToken:
+          description: >-
+            Ripetizione del token usato per disambiguare "richieste di notificazione" 
+            effetuate con lo stesso numero di protocollo (campo _paProtocolNumber_). 
+          type: string
+    
+    NewNotificationRequestStatusResponse:
+      allOf: 
```

Comment:
Perché riportare tutte le informazioni della notifica che il caso d'uso è monitorare lo stato della richiesta e recuperare lo IUN?

Reply:
E' l'unico modo che la PA ha di fare troubleshooting in caso di Richiesta rifiutata (la richiesta rifiutata non ha lo IUN quindi non può essere recuperata tramite una "getByIUN"